src/views/RepoHeaderLabelsView/InfluenceChart.tsx

```diff
@@ -2,67 +2,72 @@ import React, { useEffect, useRef } from 'react';
 import * as echarts from 'echarts';
 import { formatNum, numberWithCommas } from '../../utils/formatter';
 
-const COLORS = {
+const LIGHT_THEME = {
+  FG_COLOR: '#24292F',
+  BG_COLOR: '#ffffff',
+  SPLIT_LINE_COLOR: '#D0D7DE',
+  BAR_COLOR: '#ED3E4A',
+  LINE_COLOR: '#F02331',
+};
+
+const DARK_THEME = {
   FG_COLOR: '#c9d1d9',
   BG_COLOR: '#0d1118',
-  SPLIT_LINE: 'grey',
-  PALLET: ['#3fb950'],
+  SPLIT_LINE_COLOR: '#30363D',
+  BAR_COLOR: '#ED3E4A',
+  LINE_COLOR: '#FF8088',
 };
 
 interface InfluenceChartProps {
+  theme: 'light' | 'dark';
   width: number;
   height: number;
   data: [string, number][];
 }
 
 const InfluenceChart: React.FC<InfluenceChartProps> = (props) => {
-  const { width, height, data } = props;
+  const { theme, width, height, data } = props;
 
   const divEL = useRef(null);
 
+  const TH = theme == 'light' ? LIGHT_THEME : DARK_THEME;
+
   const option: echarts.EChartsOption = {
-    color: COLORS.PALLET,
-    title: {
-      text: 'Influence',
-      textStyle: {
-        fontSize: 14,
-        color: COLORS.FG_COLOR,
-      },
-    },
     tooltip: {
       trigger: 'axis',
       textStyle: {
-        color: COLORS.FG_COLOR,
+        color: TH.FG_COLOR,
       },
-      backgroundColor: COLORS.BG_COLOR,
+      backgroundColor: TH.BG_COLOR,
       formatter: tooltipFormatter,
     },
     grid: {
-      top: '20%',
-      left: '12%',
-      width: '85%',
-      height: '60%',
+      top: '5%',
+      bottom: '5%',
+      left: '5%',
+      right: '5%',
+      containLabel: true,
```

Comment:
Thanks @pissang for pointing out the `containLabel` option to make the chart filling its parent container much more easier. This magic option saves my life and I think everyone who uses ECharts should know it :)